tdvt/TDVT Results.twb

```diff
@@ -59,20 +60,60 @@
-          <columns character-set='UTF-8' header='yes' locale='en_US' separator='&#9;'>
-            <column datatype='string' name='Test Function' ordinal='0' />
-            <column datatype='string' name='Test Category' ordinal='1' />
-            <column datatype='string' name='Test Path Categorized' ordinal='2' />
-            <column datatype='string' name='Test Name Categorized' ordinal='3' />
-            <column datatype='string' name='Function Categorized' ordinal='4' />
-            <column datatype='string' name='Priority' ordinal='5' />
-          </columns>
+        <_.fcp.ObjectModelEncapsulateLegacy.false...relation join='inner' type='join'>
+          <clause type='join'>
+            <expression op='='>
+              <expression op='[test_results_combined.csv].[TestName]' />
+              <expression op='[test_metadata.csv].[Test Name Categorized]' />
+            </expression>
+          </clause>
+          <relation connection='textscan.1gc4mna00d9z6u19vsqbk09t94y2leaf' name='test_results_combined.csv' table='[test_results_combined#csv]' type='table'>
+            <columns character-set='UTF-8' header='yes' locale='en_US' separator=',' text-qualifier='&quot;'>
+              <column datatype='string' name='Suite' ordinal='0' />
+              <column datatype='string' name='Test Set' ordinal='1' />
+              <column datatype='string' name='TDSName' ordinal='2' />
+              <column datatype='string' name='TestName' ordinal='3' />
+              <column datatype='string' name='TestPath' ordinal='4' />
+              <column datatype='boolean' name='Passed' ordinal='5' />
+              <column datatype='integer' name='Closest Expected' ordinal='6' />
+              <column datatype='integer' name='Diff count' ordinal='7' />
+              <column datatype='string' name='Test Case' ordinal='8' />
+              <column datatype='string' name='Test Type' ordinal='9' />
+              <column datatype='string' name='Priority' ordinal='10' />
+              <column datatype='string' name='Categories' ordinal='11' />
+              <column datatype='string' name='Functions' ordinal='12' />
+              <column datatype='string' name='Process Output' ordinal='13' />
+              <column datatype='string' name='Error Msg' ordinal='14' />
+              <column datatype='string' name='Error Type' ordinal='15' />
+              <column datatype='real' name='Query Time (ms)' ordinal='16' />
+              <column datatype='string' name='Generated SQL' ordinal='17' />
+              <column datatype='string' name='Actual (100)tuples' ordinal='18' />
+              <column datatype='string' name='Expected (100)tuples' ordinal='19' />
+            </columns>
+          </relation>
+          <relation connection='textscan.0j1revp1n7tkl2195svbx0pfss2u' name='test_metadata.csv' table='[test_metadata#csv]' type='table'>
+            <columns character-set='UTF-8' header='yes' locale='en_US' separator=','>

```

Comment:
This now has the wrong separator value for the `test_metadata.csv` file. This should be reverted to tab, not comma.